models/cocktail-model.js

```diff
@@ -1,8 +1,12 @@
 const mysql = require('./connection');
-
+exports.getCocktailById = async (id) => {
+    const connection = await mysql.connect();
+    const [data] = await connection.query(`SELECT * FROM cocktails WHERE id = ?` , id)
```

Comment:
there should be a semicolon after ? to prevent injection. the code may work without it but not in every instance.